### PR TITLE
improved the background of the login page 

### LINKED ISSUE #834: ✨[UI Enhancement]: I would like to enhance the ui of the login page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

i would like to enhance the ui of the login page

### Use Case

Before : 
image

After : 
image
I will make some more changes but this is the first change i made to the ui i will also change the design a little of the containers
This is going to be a moving background which will be a little blurry so that the containers are more visible

### Benefits

_No response_

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by aditya-bhaumik**
## Description

i have added hover effect to the social icons and the home button 
I have added a linear gradient background 
i have added a wave animation to the new linear background 

## Related Issues

[Cite any related issue(s) this pull request addresses. If none, simply state “None”]
- Closes #834 

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / videos (if applicable)
Before :

image

After : 

image

The background is continuously moving

## Checklist

- [ ] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [ ] I have updated my branch and synced it with project `main` branch before making this PR
- [ ] I have performed a self-review of my code
- [ ] I have tested the changes thoroughly before submitting this pull request.
- [ ] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ ] I have commented my code, particularly in hard-to-understand areas.

## Additional context:
[Include any additional information or context that might be helpful for reviewers.]

### CODE REVIEW

**Comment by aditya-bhaumik** [comment]
Please review my pull request @varshith257 @Anishkagupta04

**Comment by varshith257** [comment]
Hey @aditya-bhaumik , it's good with it's color scheme matching with project. Can you remove the animation and keep static color matching with color scheme ?

You can also make it mobile responsive it's looking really bad in mobile view of login/sign up page

**Comment by aditya-bhaumik** [comment]
@varshith257 Sure, I’ll make the changes and let you know as soon as I can.

**Comment by aditya-bhaumik** [comment]
@varshith257 i have made the background static and now the page is more responsive then before 
image

image

image

image

**Comment by varshith257** [comment]
See here some line is appearing in the middle of page . I think it is due to animation

**Comment by varshith257** [comment]
For mobile response I think we have to make text size small and same buttons and social media icons to better align with mobile screen

**Comment by aditya-bhaumik** [comment]
@varshith257 now the line should not be coming i made the text and all small can you check again for different devices

**Comment by aditya-bhaumik** [comment]
i checked in mine for different devices and it is looking good in all of them

image
image
image

**Comment by varshith257** [comment]
> i checked in mine for different devices and it is looking good in all of them
> 
> image

Exactly here I think we have to define multiple media screens if needed to inline the remembere and forgot password could align in one line

**Comment by aditya-bhaumik** [comment]
@varshith257 hi i tried align the 2 text in same line but no matter how many media queries i add there are some screens where these 2 are not coming in same line

**Comment by varshith257** [comment]
@aditya-bhaumik Have you tried with reducing text sizes where isn't aligning

**Comment by aditya-bhaumik** [comment]
@varshith257 yes it is just make text smaller it is not getting them in the same line

**Comment by aditya-bhaumik** [comment]
image
see like this

File: log/login.css [hunk 1]
```diff
@@ -341,4 +342,112 @@ input {
...
+  }
+
+  button {
+    padding: 10px 30px;
+    font-size: 1rem;
+  }
+}
+
+/* Adjustments for screens smaller than 768px */
+@media (max-width: 768px) {
```
Comment (varshith257) [inline log/login.css:375]: Use <a> and <label>  tags here to reduce their sizes as how remember me and forgot password are aligned in login.html

And also move the login.html page here to log folder 

login.html has multiple <script> tags (duplicate) remove them too

**Review by varshith257 — CHANGES_REQUESTED** [review]

**Comment by aditya-bhaumik** [comment]
@varshith257 i added the login.html to the log folder and removed the duplicate script code i also tried the labels you mentioned but the result is same as before

**Review by varshith257 — APPROVED** [review]

**Comment by varshith257** [comment]
Ok, let's come to it later. I have no time to see what's the cause of it

**Comment by aditya-bhaumik** [comment]
@varshith257 There was just a small change left that I was making before the code got merged. In the index.html, the link was supposed to be log/login.html to link the login page can you please make that small change

**Comment by varshith257** [comment]
Submit follow up PR with this change

**Comment by aditya-bhaumik** [comment]
@varshith257 added it in #855

**Pull request merged**